react-main/src/app/components/Login/layout.js

```diff
@@ -13,8 +16,9 @@ function required(value) {
   return value ? undefined : ' Required';
 }
 
-function FormInput(props) {
-  const validate = [required, minLength8];
+const validate = [required, minLength8];
```

Comment:
we usually put this methods in an utils.js or constants.js inside the component's folder, so the funcitonal component has only the rendering code.